addons/crm/data/ir_action_data.xml

```diff
@@ -10,12 +10,7 @@
         <field name="binding_model_id" ref="crm.model_crm_lead"/>
         <field name="binding_view_types">list</field>
         <field name="state">code</field>
-        <field name="code">
-if record:
-    action_values = env.ref('crm.crm_lead_lost_action').sudo().read()[0]
-    action_values.update({'context': env.context})
-    action = action_values
-        </field>
+        <field name="code">action = model.action_mark_as_lost()</field>
```

Comment:
Would it be worth to do a force_update on upgrade script ?

Reply:
Mm, I did now but actually I wonder. Do we need force_update for this? It doesn't seem to be marked as noupdate to begin with. Unless actions are always noupdate?

I could be missing the point of force_update though..